Python/Image_Sketch/README.md

```diff
@@ -0,0 +1,31 @@
+# Image to Sketch
+
+### Introduction
+This script converts original image into black and white sketched image.
+
+## Third Party Libraries Required :
+1. opencv library 
+2. scikit-image library
+
+### How to install above Library
+Install the following libraries through command line:
+```
+pip install opencv-python
+pip install numpy
+pip install scikit-image
+```
+### How to use it :
+1. Download or clone the repository
+2. Install required libraries
+3. Add path of your image in ```img = io.imread("enter your image path") ``` function from **Sketch.py**
```

Comment:
you need to add functionality to add image path through command line

Reply:
@tejan-singh added the functionality of the user input filename through command line